notebooks/text_classification/text_classification_introspective_rationale_explainer.ipynb

```diff
@@ -62,9 +64,9 @@
    "outputs": [],
    "source": [
     "# if quick run true, skips over embedding, most of model training, and model evaluation; used to quickly test pipeline\n",
-    "QUICK_RUN = False\n",
+    "QUICK_RUN = True\n",
```

Comment:
just checking this is intended, correct?

Reply:
yes - this will make the test run quicker and any user trying to use can always flip it to to False and run the notebook.